### PR TITLE
Bug fixing: adding the threshold evaluations and leftout topic parame…

### PR DESCRIPTION
**Pull request by mmerce**
…ters

### CODE REVIEW

File: bigmler/resources.py [hunk 1]
```diff
@@ -1098,6 +1098,16 @@ def set_evaluation_args(args, fields=None,
     if hasattr(args, 'method') and (args.number_of_models > 1
                                     or args.ensemble):
         evaluation_args.update(combiner=args.method)
+    if hasattr(args, 'method') and args.method:
+        evaluation_args.update({"combiner": args.method})
+        if args.method == 3:
```
Comment (jaor) [inline bigmler/resources.py]: it'd be better to use a named constant, rather than a raw 3

File: bigmler/resources.py [hunk 2]
```diff
@@ -1311,6 +1321,14 @@ def set_batch_prediction_args(args, fields=None,
 
     if hasattr(args, 'method') and args.method:
         batch_prediction_args.update({"combiner": args.method})
+        if args.method == 3:
```
Comment (jaor) [inline bigmler/resources.py]: same thing about the 3

File: bigmler/tests/basic_tst_prediction_steps.py [hunk 3]
```diff
@@ -458,6 +458,42 @@ def i_create_resources_from_ensemble_with_threshold(step, test=None, output2=Non
...
+                   " --remote --method threshold" +
+                   " --threshold " +
+                   str(world.number_of_models))
+        command = check_debug(command)
+        print command
+        retcode = check_call(command, shell=True)
+        if retcode < 0:
+            assert False
+        else:
+            assert True
```
Comment (jaor) [inline bigmler/tests/basic_tst_prediction_steps.py]: here we should use one of those assert functions... `assertTrue(retcode >= 0);`

File: bigmler/tests/basic_tst_prediction_steps.py [hunk 4]
```diff
@@ -458,6 +458,42 @@ def i_create_resources_from_ensemble_with_threshold(step, test=None, output2=Non
...
+                   output3 + ' --class ' + threshold_class +
+                   " --remote --method threshold" +
+                   " --threshold 1")
+        command = check_debug(command)
+        print command
+        retcode = check_call(command, shell=True)
+        if retcode < 0:
+            assert False
+        else:
+            assert True
```
Comment (jaor) [inline bigmler/tests/basic_tst_prediction_steps.py]: same thing here... `assertTrue(retcode >=0)`, or maybe there's an assertGreater, i forgot...

File: bigmler/tests/evaluation_steps.py [hunk 5]
```diff
@@ -278,3 +278,28 @@ def i_evaluate_ensemble_with_dataset(step, ensemble_dir=None, dataset_dir=None,
...
+               " --ensemble " + ensemble_id + " --store" +
+               " --output " + output + " --evaluate " + options)
+    command = check_debug(command)
+    try:
+        retcode = check_call(command, shell=True)
+        if retcode < 0:
+            assert False
+        else:
+            world.output = output
+            assert True
```
Comment (jaor) [inline bigmler/tests/evaluation_steps.py]: `assert True` does nothing, is not needed... and above `assertTrue(retcode>=0)`

**Comment by mmerce** [comment]
changes added

**Comment by mmerce** [comment]
sorry, there's one left.

**Pull request merged**